source/technical/swift.rst

```diff
@@ -6,15 +6,18 @@ ____________
 Introduction
 ____________
 
-Chameleon provides an object store service through the `OpenStack Swift <https://docs.openstack.org/swift/latest/>`_ interface. It is intended to be used for storing and retrieving data used during experiments, such as input files needed for your applications, or results produced by your experiments. 
+Chameleon provides an object store service through the `OpenStack Swift <https://docs.openstack.org/swift/latest/>`_ interface. It is intended to be used for storing and retrieving data used during experiments, such as input files needed for your applications, or results produced by your experiments.
 
 .. hint::
-   Chameleon object store service is currently backed by a `Ceph <https://ceph.com/>`_ cluster with more than 1.6 PB of capacity. The data is replicated, keeping two copies of each object, effectively providing over 800 TB of storage available to users. This storage capacity will increase as the project goes on. The replication should provide good availability in case of hardware failures. However, all copies are kept within the same data center and are not backed up on a separate system; if you feel that this does not provide sufficient reliability in your case, you should consider backing up really critical data externally.
+   Chameleon object store service is currently backed by a `Ceph <https://ceph.com/>`_ cluster with more than 2.1 PB of capacity. The data is replicated, keeping two copies of each object, effectively providing over 1 PB of storage available to users. This storage capacity will increase as the project goes on. The replication should provide good availability in case of hardware failures. However, all copies are kept within the same data center and are not backed up on a separate system; if you feel that this does not provide sufficient reliability in your case, you should consider backing up really critical data externally.
```

Comment:
Shouldn't we clearly explain that they are two separate Ceph clusters, each with their own capacity?